code/game/objects/structures/tables_racks.dm

```diff
@@ -86,6 +89,8 @@
 /obj/structure/table/examine(mob/user)
 	. = ..()
 	. += deconstruction_hints(user)
+	if(can_flip)
+		. += span_notice("You can Ctrl+Shift+Click to flip the table.")
```

Comment:
@Mothblocks didn't we decide to use contextual tips for all control mentions?

Reply:
Yes and I even said that in my feedback but they resolved it before doing it :(